src/cockpit/channels/filesystem.py

```diff
@@ -425,6 +426,14 @@ class FsInfoChannel(Channel, PathWatchListener):
 
     @staticmethod
     def make_getattrs(attrs: Iterable[str]) -> 'Callable[[int, str, Follow], JsonDocument | None]':
+        faccessat = ctypes.CDLL(None).faccessat
```

Comment:
You mention python's lack of support and /proc/self/fd/ in the commit message, then you pull out ctypes here.  Did you change your mind at some point?